application/v1alpha1/README

```diff
@@ -0,0 +1,2 @@
+The files in this directory were copied from
+https://github.com/redhat-appstudio/application-api/tree/main/api/v1alpha1
```

Comment:
As a follow up, we could probably create a little script that automates this process. Then, we can run as a GitHub Workflow, warning us if any changes are found.

Reply:
Created [HACBS-2441](https://issues.redhat.com/browse/HACBS-2441), I think we want to have an approach that is triggered on version updates and doesn't block us in CI